server/utils/db/connection_pool.py

```
@@ -161,26 +162,108 @@ def get_connection(self):
-                    with connection.cursor() as cur:  # No RLS needed — pool cleanup (RESET vars)
+                    with connection.cursor() as cur:
                         cur.execute(
-                            "RESET myapp.current_user_id; RESET myapp.current_org_id;"
+                            "RESET ROLE; "
+                            "RESET myapp.current_user_id; "
+                            "RESET myapp.current_org_id; "
+                            "RESET myapp.mcp_token_resolve; "
+                            "RESET myapp.kubectl_token_resolve;"
                         )
                     connection.commit()
                 except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)